appserver/security/core-ee/src/main/java/com/sun/enterprise/security/acl/RoleMapperFactory.java

```diff
@@ -46,73 +46,86 @@
 import java.util.HashMap;
 import java.util.Hashtable;
 
-
 import org.jvnet.hk2.annotations.Service;
 import javax.inject.Singleton;
 
 /**
  *
- * @author  Jerome Dochez
+ * @author Jerome Dochez
  */
 @Service
 @Singleton
 public class RoleMapperFactory implements SecurityRoleMapperFactory {
-    private  Map CONTEXT_TO_APPNAME = new HashMap();
-    private  Hashtable ROLEMAPPER = new Hashtable();
-    
+    private Map CONTEXT_TO_APPNAME = new HashMap();
+    private Hashtable ROLEMAPPER = new Hashtable();
+
     /** Creates a new instance of RoleMapperFactory */
     public RoleMapperFactory() {
     }
-    
-    /** Returns a RoleMapper corresponding to the AppName.
-     * @param  The Application Name of this RoleMapper.
+
+    /**
+     * Returns a RoleMapper corresponding to the AppName.
+     * 
+     * @param The
+     *            Application Name of this RoleMapper.
```

Comment:
Is there a reason why this is over two lines (and many other changes like it)?

Reply:
That's the default formatter really